pkg/support/dump_test.go

```diff
@@ -38,3 +46,38 @@ func TestParseFileName(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, ts.String(), "2021-08-17 09:47:58 +0000 UTC")
 }
+
+func TestTimestampFilter(t *testing.T) {
+	since := "7s"
+	ts := timestampFilter(since)
+	currentTime := time.Now()
+	assert.Contains(t, ts.String(), currentTime.Format("2006-01-02"))
+}
+
+func TestWriteFile(t *testing.T) {
+	fs := afero.NewOsFs()
+	resource := "networkpolicies"
+	basedir := "dir1/dir2"
+	data := []byte("antct -oyaml get networkpolicies")
+	err := writeFile(fs, filepath.Join(basedir, resource), resource, data)
+	if err != nil {
+		require.EqualError(t, err, "error when writing networkpolicies to file: open dir1\\dir2\\networkpolicies: The system cannot find the path specified.")
+	} else {
+		require.NoError(t, err)
```

Comment:
when err is nil it comes to line 66, so try to print additional informative lines for example assert.NoError(t, err, "validIPSecCSR should not return an error")